pkgs/tools/misc/goss/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "goss";
+  version = "0.3.16";
+
+  src = fetchFromGitHub {
+    owner = "aelsabbahy";
+    repo = "goss";
+    rev = "v${version}";
+    sha256 = "1m5w5vwmc9knvaihk61848rlq7qgdyylzpcwi64z84rkw8qdnj6p";
+  };
+
+  vendorSha256 = "1lyqjkwj8hybj5swyrv6357hs8sxmf4wim0c8yhfb9mv7fsxhrv7";
+
+  buildFlagsArray = [ "-ldflags=-s -w -X=main.version=${version}" ];
```

```suggestion
  preBuild = ''
    buildFlagsArray+=("-ldflags" "-s -w -X main.version=v${version}")
  '';
```

It's arguably better to set `buildFlagsArray` through bash currently, but it's not a big deal, especially for just setting ldflags.
Go packaging docs also don't reflect this suggestion atm.
More details here:
https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-flags-array.md

---

Thanks for pointing that out, I'll give `nixpkgs-hammering` a closer look :)